### PR TITLE
chore: update node agent compaitility and upgrade notes for Node agent v8 release.

### PR DESCRIPTION
**Pull request by carlo-808**
For Node agent version 8.0.0 release, this PR updates the Upgrade Node Agent and Node Agent compatibility docs.

### CODE REVIEW

File: src/content/docs/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx [hunk 1]
```diff
@@ -107,7 +120,7 @@ The following are proposed time ranges. The actual release date may vary.
       </td>
 
       <td>
-        As of July 31, 2021, we're discontinuing support for Node.js 10. For more details, including how you can easily prepare for this transition, see our [Explorers Hub post](https://discuss.newrelic.com/t/important-upcoming-changes-to-support-and-capabilities-across-browser-node-js-agent-query-builder-net-agent-apm-errors-distributed-tracing/153373).
+        As of July 26, 2021, we're discontinuing support for Node.js 10. For more details, including how you can easily prepare for this transition, see our [Explorers Hub post](https://discuss.newrelic.com/t/important-upcoming-changes-to-support-and-capabilities-across-browser-node-js-agent-query-builder-net-agent-apm-errors-distributed-tracing/153373).
```
Comment (michaelgoin) [inline src/content/docs/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx:123]:
```suggestion
        As of July 26, 2021, we have discontinued support for Node.js 10 with v8 of the Node.js agent. For more details, including how you can easily prepare for this transition, see our [Explorers Hub post](https://discuss.newrelic.com/t/important-upcoming-changes-to-support-and-capabilities-across-browser-node-js-agent-query-builder-net-agent-apm-errors-distributed-tracing/153373).
```

File: src/content/docs/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx [hunk 2]
```diff
@@ -52,7 +65,7 @@ The following are proposed time ranges. The actual release date may vary.
       </td>
 
       <td>
-        April-October 2021
+        July 26, 2021 with Node Agent v8.0.0
```
Comment (michaelgoin) [inline src/content/docs/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx:68]:
```suggestion
        July 26, 2021 with Node.js agent v8.0.0
```

**Comment by zuluecho9** [comment]
@carlo-808 Hi Carlo. This PR is set to draft. Is that correct? Or can this go out any time? (because the associated release note I'm trying to publish now.)

File: src/content/docs/agents/nodejs-agent/installation-configuration/update-nodejs-agent.mdx [hunk 3]
```diff
@@ -18,6 +18,94 @@ To take full advantage of New Relic's latest features, enhancements, and importa
...
+</table>
+
+## Node version support [#node-support-v8]
+
+Node v12 is the earliest version supported by the New Relic Node.js v8 agent. Node 10 and 11 are not supported by v8. Customers running Node 11 and earlier have two options:
+
+* Upgrade to a supported version of Node and take advantage of the New Relic Node.js v8 agent's new features.
+* Remain on New Relic Node.js v6 agent without the ability to use new features only available with update agent versions.
+
+[Node 12 no longer receive updates](https://github.com/nodejs/Release/tree/7a62313a0cf40e3fd6c75c3643b2637289acc82d), but New Relic will continue to support these versions of Node for the time being.
```
Comment (michaelgoin) [inline src/content/docs/agents/nodejs-agent/installation-configuration/update-nodejs-agent.mdx:103]: Is this true? I don't think that is true.

**Comment by carlo-808** [comment]
> @carlo-808 Hi Carlo. This PR is set to draft. Is that correct? Or can this go out any time? (because the associated release note I'm trying to publish now.)

This is a draft and I'm updating some of the docs based on a review. That being said, I'll probably set the PR to review soon.

**Comment by carlo-808** [comment]
OK, @zuluecho9 , the PR is ready.

**Review by zuluecho9 — APPROVED** [review]

**Pull request merged**